inc/integrations/class-pico.php

```diff
@@ -77,6 +78,15 @@ public function register_settings_fields() {
 			'wp_irving_integrations',
 			'irving_integrations_settings'
 		);
+
+		// Register new fields for the Coral integration.
+		add_settings_field(
+			'wp_irving_pico_tiers',
+			esc_html__( 'Pico/Coral Whitelisted SSO Tiers', 'wp-irving' ),
```

Comment:
Avoid "Whitelist" in favor of more inclusive language, here and throughout 🔢 

```suggestion
			esc_html__( 'Allowed Pico/Coral SSO Tiers', 'wp-irving' ),
```

Reply:
Great point! I hadn't even considered the terms lack of inclusivity.